src/core/catalog.js

```diff
@@ -642,6 +643,32 @@ class Catalog {
       parsedOrderRefs = new RefSet(),
       MAX_NESTED_LEVELS = 10;
 
+    // Parse RBGroups entry.
+    (rbGroups => {
```

Comment:
Because the function is local + only called once while loading a PDF and its return value is not used, I thought an IIFE would be in order. However, I can change it back to a named function that is called subsequently.